### ISSUE TITLE
Use case: Display a location on a map

### ISSUE THREAD

**Issue by AmeliaBR** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#33]
This issue is for discussion of [the use case “Display a specific location on an interactive map”](https://maps4html.github.io/HTML-Map-Element-UseCases-Requirements/#use-case-single-location), its examples & list of required capabilities.

This is currently the first use case listed, because I think it is the simplest/most common use case on the web.

**Comment by prushforth** [comment]
This use case has many aspects to consider. First, what is a location? That is a profound subject, but fortunately, there is a comprehensive standard, known as the Simple Feature model, upon which many formats and services are based, including GeoJSON, Shapefiles, KML, GML etc.

Next thing to consider is what does it mean to 'display a location on a map', in particular for blind or low-vision users? I believe a user agent should be able to provide an answer, although I am not an expert in this area. Ideally others can help us out here.

Finally, the map. A location can (should / does) have a different representation at different scales. The map therefore needs the concept of scale. The de facto proxy for scale that has arisen on the Web, especially because of the use of tile services, is 'zoom level'. 

There are probably lots more things to add. I'll try to add them as I think of them.

**Comment by AmeliaBR** [comment]
Scale is a very good point.  A map centered on Portage & Main, Winnipeg, is going to look very different depending on whether it's zoomed in to the city blocks or zoomed out to all of North America.  The use cases for those maps are very different.

Another way to think about it is to separate out the idea of "displaying an *area* (e.g., bounding box)" on a map from the idea of displaying pinpoint markers (which will also be covered in use case #34 ).

**Comment by prushforth** [comment]
Different types of geometry are certainly worth discussing, but I think the most important aspect of this use case is not the display, but the accessibility / machine interpretabilty of the display, in the sense of [geofencing](https://en.m.wikipedia.org/wiki/Geo-fence). That article is not great, especially because it is region-centric, but it may lead us to what I'm trying to say: a website should be able to transmit a representation of a feature that can be used by the user agent to tell the user about the near-ness or in-ness of a location (sometimes importantly the device location, but not exclusively) to the feature that is 'displayed' (as in the title of this use case). So its not exclusively an accessibility feature, but it could be extremely useful to blind or low-vision users, which may be any of us, situationally speaking.

**Comment by AmeliaBR** [comment]
Comparing the location/region on the map to the user's current location (and maybe generating descriptions) is a valid use case/capability, but that's separate from just displaying the map.

But your general point about alternative text for the map itself — to provide non-visual users the information that a visual user receives from looking at the rendering — is important to discuss.  It's complicated enough that I might want to make it a separate use case. (Even though I said in #39 that I wanted to integrate accessibility into the primary use cases when I came to interaction.)

**Comment by prushforth** [comment]
>Comparing the location/region on the map to the user's current location (and maybe generating descriptions) is a valid use case/capability, but that's separate from just displaying the map.

Yes, I agree.  However, this use case blends these two situations, and probably reasonably so, because a location without an underlying map is visually useless (or nearly so) and displaying the map with only a visual component is useless (or nearly so) to non-visual users.

Again the user's current location isn't necessarily in play (although naturally we thing of our GPS locatin as the centre of the universe); the centre of the map display (having been panned and or zoomed) may be the point of comparison, for example, to the Web site's displayed location (from the title of this use case).  For that matter, the location of interest might itself not be a point, it could be a street or a lake: how far is the 'location displayed on the map' from street x, etc. 

Mostly what I'm trying to get at here is that locations need to be 'semantic', unlike for example in SVG (no insult intended, just discussing) where a path can represent anything, and further, it's not a big deal if you have to duplicate paths for symbolization.

**Comment by hannesaddec** [comment]
Following Daniel's suggestion at FOSS4G2019 I'm commenting here. In the EC we have an inhouse corporate level service WEBTOOLS - which has a map component. In there, the [contact map](https://europa.eu/webtools/showcase/demo/cmap/2.0/) is used to visualize a point in a map. It's quite popular  as it's a simple way to show a single(or several locations). This comment also belongs to #77. I welcome this initiative.
